jupiterone/resource_rule.go

```diff
@@ -351,6 +351,14 @@ func (*QuestionRuleResource) ModifyPlan(ctx context.Context, req resource.Modify
 		return
 	}
 
+	// switching from referenced to inline requires replacement
+	if plan.QuestionId.IsNull() && !state.QuestionId.IsNull() {
+		resp.RequiresReplace = append(resp.RequiresReplace, path.Root("question_id"))
+	}
+	if len(plan.Question) == 0 && len(state.Question) != 0 {
+		resp.RequiresReplace = append(resp.RequiresReplace, path.Root("question"))
+	}
```

Comment:
I talked with the team that maintains rules, and the error mentioned above seems to be a bug that we'll get fixed. It will be possible to switch between referenced and in-line rules. I think we can remove this part.

Reply:
Done. The tests will need to be updated/regenerated when the bug is fixed, so I have opened #190 to keep track of it.